<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>transaction</class>
 <widget class="QDialog" name="transaction">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="btn_prev">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>950</y>
     <width>381</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/icons/flags/hemmobutt.png);
font: 900 34pt &quot;Comic Sans MS&quot;;
border-style: solid;
border-color: white;
border-width: 2px;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>Edellinen</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonExit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>950</y>
     <width>381</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/icons/flags/hemmobutt.png);
font: 900 34pt &quot;Comic Sans MS&quot;;
border-style: solid;
border-color: white;
border-width: 2px;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>Takaisin</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_next">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>950</y>
     <width>381</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/icons/flags/hemmobutt.png);
font: 900 34pt &quot;Comic Sans MS&quot;;
border-style: solid;
border-color: white;
border-width: 2px;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>Seuraava</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>480</y>
     <width>1141</width>
     <height>441</height>
    </rect>
   </property>
   <widget class="QWidget" name="page_7"/>
   <widget class="QWidget" name="page_8">
    <widget class="QTextEdit" name="TransactionsPrevious">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>501</width>
       <height>241</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label_username">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>1161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>22</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/icons/flags/hemmobutt.png);
font: 900 22pt &quot;Comic Sans MS&quot;;
border-style: solid;
border-color: white;
border-width: 2px;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>label4</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="Transactions">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>480</y>
     <width>1161</width>
     <height>441</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/icons/flags/hemmobutt.png);
font: 900 18pt &quot;Comic Sans MS&quot;;
border-style: solid;
border-color: white;
border-width: 2px;
border-radius: 10px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="backroundPic">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>1080</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resourcefile.qrc">:/icons/flags/hemmotransactions.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>1217</x>
     <y>540</y>
     <width>311</width>
     <height>144</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>34</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/icons/flags/hemmobutt.png);
font: 900 34pt &quot;Comic Sans MS&quot;;
border-style: solid;
border-color: white;
border-width: 2px;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>Tilinumerosi:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_accountnumber">
   <property name="geometry">
    <rect>
     <x>1540</x>
     <y>540</y>
     <width>391</width>
     <height>144</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>22</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/icons/flags/hemmobutt.png);
font: 900 22pt &quot;Comic Sans MS&quot;;
border-style: solid;
border-color: white;
border-width: 2px;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>accNum</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>1215</x>
     <y>700</y>
     <width>311</width>
     <height>143</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>33</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/icons/flags/hemmobutt.png);
font: 900 33pt &quot;Comic Sans MS&quot;;
border-style: solid;
border-color: white;
border-width: 2px;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>Luottorajasi:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_credit">
   <property name="geometry">
    <rect>
     <x>1540</x>
     <y>700</y>
     <width>391</width>
     <height>143</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>34</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/icons/flags/hemmobutt.png);
font: 900 34pt &quot;Comic Sans MS&quot;;
border-style: solid;
border-color: white;
border-width: 2px;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>credit</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>1215</x>
     <y>860</y>
     <width>311</width>
     <height>144</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>34</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/icons/flags/hemmobutt.png);
font: 900 34pt &quot;Comic Sans MS&quot;;
border-style: solid;
border-color: white;
border-width: 2px;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>Saldosi:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_saldo">
   <property name="geometry">
    <rect>
     <x>1540</x>
     <y>860</y>
     <width>391</width>
     <height>144</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>34</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/icons/flags/hemmobutt.png);
font: 900 34pt &quot;Comic Sans MS&quot;;
border-style: solid;
border-color: white;
border-width: 2px;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>saldo</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>480</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 900 24pt &quot;Comic Sans MS&quot;;
</string>
   </property>
   <property name="text">
    <string>Määrä:</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>480</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 900 24pt &quot;Comic Sans MS&quot;;
</string>
   </property>
   <property name="text">
    <string>Sijainti:</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>480</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 900 24pt &quot;Comic Sans MS&quot;;
</string>
   </property>
   <property name="text">
    <string>Päivämäärä:</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtEdit_balanceChange">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>530</y>
     <width>241</width>
     <height>391</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/icons/flags/hemmobutt.png);
font: 900 34pt &quot;Comic Sans MS&quot;;

border-color: white;
border-width: 2px;
border-radius: 10px;
</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtEdit_location">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>530</y>
     <width>281</width>
     <height>391</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/icons/flags/hemmobutt.png);
font: 900 34pt &quot;Comic Sans MS&quot;;

border-color: white;
border-width: 2px;
border-radius: 10px;
</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtEdit_date">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>530</y>
     <width>591</width>
     <height>391</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/icons/flags/hemmobutt.png);
font: 900 34pt &quot;Comic Sans MS&quot;;

border-color: white;
border-width: 2px;
border-radius: 10px;
</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Comic Sans MS'; font-size:34pt; font-weight:900; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Segoe UI'; font-size:9pt; font-weight:400;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_pageNUM">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>480</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 900 24pt &quot;Comic Sans MS&quot;;
</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>480</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 900 24pt &quot;Comic Sans MS&quot;;
</string>
   </property>
   <property name="text">
    <string>/ 20</string>
   </property>
  </widget>
  <zorder>backroundPic</zorder>
  <zorder>btn_prev</zorder>
  <zorder>ButtonExit</zorder>
  <zorder>btn_next</zorder>
  <zorder>stackedWidget</zorder>
  <zorder>label_username</zorder>
  <zorder>Transactions</zorder>
  <zorder>label_3</zorder>
  <zorder>label_accountnumber</zorder>
  <zorder>label_2</zorder>
  <zorder>label_credit</zorder>
  <zorder>label</zorder>
  <zorder>label_saldo</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>label_6</zorder>
  <zorder>txtEdit_balanceChange</zorder>
  <zorder>txtEdit_location</zorder>
  <zorder>txtEdit_date</zorder>
  <zorder>label_pageNUM</zorder>
  <zorder>label_8</zorder>
 </widget>
 <layoutdefault spacing="6" margin="9"/>
 <resources>
  <include location="resourcefile.qrc"/>
 </resources>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
